openpocket/alpha_spheres/alpha_spheres.py

```diff
@@ -126,43 +126,44 @@ def remove_alpha_spheres(self, indices):
 
         """
         if indices is not None:
-        #    if isinstance(indices, (list,np.ndarray,dtype:ints)): 
-            if isinstance(indices, (list,np.ndarray)):
+            if isinstance(indices, (list,np.ndarray,tuple)):
                 pass
             else:
-                raise ValueError("The argument points needs to be a List, tuple or numpy.ndarray object with the integer numbers corresponding to the alpha-sphere")
-
-            self.indices = indices  # a way to store the input indices
-            alpha_sphere_index=[]
-            for ia,alpsp in enumerate(self.points_of_alpha_sphere):
-                alpha_sphere_index.append(ia) 
-            for ix in self.indices:
-#                if ix in alpha_sphere_index:
-                alpha_sphere_index.remove(ix)
-               # remaining=[]
-                self.remove_alpha_spheres=[]
-                for ai in alpha_sphere_index:
-                    self.remove_alpha_spheres.append(self.points_of_alpha_sphere[ai])
-               #     self.remove_alpha_spheres = self.points_of_alpha_sphere[ai]
-               # else:
-               #     raise IndexError("has been using a non existent index")
-
+                raise ValueError("The input indices needs to be a List, tuple or numpy.ndarray object with the integer numbers corresponding to the alpha-sphere")
+            for ix in indices:
+                point_index.remove(ix)
+            
+            remaining_centers=[]
+            remainig_points_of_alpha_sphere=[]
+            remainig_points=[]
+            for ri in point_index:
```

Comment:
Shouldn't this be
```
for ri in indices:
```
?

Reply:
I think it could work in the way that is writed; because the method remove, removes the input indices of the point_index, so  the resulting point_index must have the remainig indices, that's why I choose that variable.